GalaxyZooTouchTable/GalaxyZooTouchTable/Models/GalaxyRing.cs

```diff
@@ -68,21 +70,21 @@ private void PlaceAvatar()
             {
                 switch (UserName)
                 {
-                    case "HeartUser":
+                    case "PinkUser":
                         AvatarY *= -1;
                         AvatarX *= -1;
                         break;
-                    case "StarUser":
+                    case "BlueUser":
                         AvatarY *= -1;
                         break;
-                    case "LightUser":
+                    case "AquaUser":
                         AvatarY = 0;
                         AvatarX = Diameter / 2;
                         break;
-                    case "PersonUser":
+                    case "PurpleUser":
                         AvatarX *= -1;
                         break;
-                    case "EarthUser":
+                    case "GreenUser":
```

Comment:
We have a relatively large case statement--which is OK, since it's pretty clear what it does.

Another option might be to put attributes for `XPosition` and `YPosition` onto the `user` object that we're passing into `GalaxyRing`,  and then assign `AvatarX` and `AvatarY` to those attributes inside of `PlaceAvatar()`.

Reply:
One issue with that is that a ring's `AvatarX` and `AvatarY` will change values whenever a user leaves the table and their ring around a subject disappears. Thus, the X and Y positions are changing depending on how many users have classified a subject and in which order (index value) the user's ring appears. 

However, most of this display stuff probably shouldn't be done in the Models, so I think your point is valid that there is probably a better solution to be had. I'll timebox this and see if I can find an alternative.